tsconfig.json

```diff
@@ -114,5 +114,9 @@
         },
         "jsx": "react-jsx"
     },
-    "include": ["resources/js/**/*.ts", "resources/js/**/*.tsx"]
+    "include": [
+        "resources/js/**/*.ts",
+        "resources/js/**/*.d.ts",
```

Comment:
Would "resources/js/\*\*/\*.d.ts" not already be targeted with "resources/js/\*\*/\*.ts"?

Reply:
I don't think it is necessary, but I feel like it's not a bad thing to include because it helps with clarity. Some devs perfer to explicitly include the `.d.ts` 👍